html/janus.js

```diff
@@ -949,6 +949,7 @@ function Janus(gatewayCallbacks) {
 		callbacks = callbacks || {};
 		// FIXME This method triggers a success even when we fail
 		callbacks.success = (typeof callbacks.success == "function") ? callbacks.success : Janus.noop;
+                callbacks.error = (typeof callbacks.error == "function") ? callbacks.error : Janus.noop;
```

Comment:
Please fix the indentation: we use tabs, not spaces.

Reply:
Ok done.

Reply:
Thx :+1: As soon as the CLA is signed this is good to merge, then.